libsolidity/ast/Types.cpp

```diff
@@ -1527,8 +1529,6 @@ string ReferenceType::identifierLocationSuffix() const
 		id += "_calldata";
 		break;
 	}
-	if (isPointer())
-		id += "_ptr";
```

Comment:
Is this breaking? I could also *always* append ``_ptr`` for memory types, but both variants change the JSON ASTs one way or another...

Reply:
This changes calldata, too. I think I'll revert this and only change the constructor.